components/automate-ui/src/app/pages/automate-settings/automate-settings.component.scss

```diff
@@ -10,8 +10,9 @@ chef-td {
   }
 }
 
-chef-input {
+input {
   width: 60px;
+  height: 100%;
```

Comment:
this height was needed in order to keep the inputs on this form all uniform size.